packages/components/src/range-control/styles/range-control-styles.ts

```diff
@@ -139,28 +143,18 @@ export const MarksWrapper = styled.span`
 	position: relative;
 	width: 100%;
 	user-select: none;
+	margin-top: 17px;
 `;
 
-const markFill = ( { disabled, isFilled }: RangeMarkProps ) => {
-	let backgroundColor = isFilled ? 'currentColor' : COLORS.gray[ 300 ];
-
-	if ( disabled ) {
-		backgroundColor = COLORS.gray[ 400 ];
-	}
-
-	return css( {
-		backgroundColor,
-	} );
-};
-
 export const Mark = styled.span`
-	height: ${ thumbSize }px;
-	left: 0;
 	position: absolute;
-	top: 9px;
-	width: 1px;
-
-	${ markFill };
+	left: 0;
+	top: -4px;
+	height: 4px;
+	width: 2px;
+	transform: translateX( -50% );
+	background-color: ${ COLORS.white };
```

Comment:
This prompted me to deprecate the `COLORS.white` variable (#67649). Here's a good replacement:

```suggestion
	background-color: ${ COLORS.ui.background };
```

Reply:
Is there any interest in keeping this consistent with the CSS variables (I believe these variables originate from there)

Reply:
Do you mean like the `--wp-admin-theme-color-*` CSS variables, or the Sass variables in base-styles colors?

Currently we are still at the stage where we're validating the set of collectively exhaustive color tokens to support themability, so we're not ready to commit to anything back to the upstream color variables yet.

Reply:
The way I see it is:

> --wp-admin-theme-color-* CSS variables 

These should just be the "theme" object we always talked about

> Sass variables

These are the same as these utils we use in several components but we don't want them to be considered as a "theme" config.

Reply:
"white" for me falls in the second set.

Reply:
Not sure I understand what you're getting at, is there a specific action item you are proposing?

If you're saying that `$white` is generally not a valid token in a themeable system and shouldn't be in base-styles colors, then yes I agree. I assume we will get there eventually.

Reply:
Yeah, I think we agree. What I'm saying is that even the UI background color might not be a good token for a theme but nothing actionable for now.

Reply:
And we'll definitely clarify in the end product (or maybe rename), but "background" in this context refers to what background color the component is intended to be rendered on top of, not necessarily the background color within the component itself.